lua_modules/ftp/server.lua

```diff
@@ -0,0 +1,157 @@
+-- a simple ftp server
+USER = "test"
+PASS = "12345"
```

Comment:
Keep these in locals, and they will be accesses upvals in nested closures.  See my FAQ for moy explanation:
```Lua
local USER, PASS = "test", "12345"
```